docs/yourbase-ruby.md

```diff
@@ -0,0 +1,73 @@
+# YourBase Ruby Acceleration
+
+Tests are important. For large monoliths, they're also a major source of drag on velocity.
+
+YourBase is a tool that traces your tests to determine which functions each test depends on. It can later use this information to determine which tests do not need to run because their code paths have not changed. These tests are skipped automatically.
```

Comment:
```suggestion
YourBase is a tool that traces your tests to determine which files they depend on. It uses this information to determine which tests must run, because they are new or have changed dependencies. Tests without changed dependencies are skipped.
```

Reply:
Thx, copy and paste from Python was a bad idea here.  :)